lib/common/hash.go

```diff
@@ -64,6 +64,12 @@ func (h Hash) String() string {
 	return fmt.Sprintf("0x%x", h[:])
 }
 
+// Short returns the first 3 bytes and the last 3 bytes of the hex string for the hash
+func (h Hash) Short() string {
+	const nBytes = 3
```

Comment:
👍 Might be useful!
Although just in case I'd just have 4+4 characters, since 16^6 = 16,777,216‬ and 16^8 = 4,294,967,296 which is quite more collision resistant.

```suggestion
// Short returns the first 4 bytes and the last 4 bytes of the hex string for the hash
func (h Hash) Short() string {
	const nBytes = 4
```